bdk-ffi/src/keys.rs

```diff
@@ -26,91 +26,91 @@ impl Mnemonic {
     /// Entropy must be a multiple of 32 bits (4 bytes) and 128-256 bits in length.
     pub(crate) fn from_entropy(entropy: Vec<u8>) -> Result<Self, BdkError> {
         BdkMnemonic::from_entropy(entropy.as_slice())
-            .map(|m| Mnemonic { internal: m })
+            .map(|m| Mnemonic { inner: m })
             .map_err(|e| BdkError::Generic(e.to_string()))
     }
 
     /// Returns Mnemonic as string
     pub(crate) fn as_string(&self) -> String {
-        self.internal.to_string()
+        self.inner.to_string()
     }
 }
 
 pub(crate) struct DerivationPath {
-    derivation_path_mutex: Mutex<BdkDerivationPath>,
+    inner_mutex: Mutex<BdkDerivationPath>,
 }
 
 impl DerivationPath {
     pub(crate) fn new(path: String) -> Result<Self, BdkError> {
         BdkDerivationPath::from_str(&path)
             .map(|x| DerivationPath {
-                derivation_path_mutex: Mutex::new(x),
+                inner_mutex: Mutex::new(x),
             })
             .map_err(|e| BdkError::Generic(e.to_string()))
     }
 }
 
 #[derive(Debug)]
 pub(crate) struct DescriptorSecretKey {
-    pub(crate) descriptor_secret_key_mutex: Mutex<BdkDescriptorSecretKey>,
+    pub(crate) inner_mutex: Mutex<BdkDescriptorSecretKey>,
 }
 
 impl DescriptorSecretKey {
     pub(crate) fn new(network: Network, mnemonic: Arc<Mnemonic>, password: Option<String>) -> Self {
-        let mnemonic = mnemonic.internal.clone();
+        let mnemonic = mnemonic.inner.clone();
         let xkey: ExtendedKey = (mnemonic, password).into_extended_key().unwrap();
-        let descriptor_secret_key = BdkDescriptorSecretKey::XPrv(DescriptorXKey {
+        let inner = BdkDescriptorSecretKey::XPrv(DescriptorXKey {
             origin: None,
             xkey: xkey.into_xprv(network).unwrap(),
             derivation_path: BdkDerivationPath::master(),
             wildcard: bdk::descriptor::Wildcard::Unhardened,
         });
         Self {
-            descriptor_secret_key_mutex: Mutex::new(descriptor_secret_key),
+            inner_mutex: Mutex::new(inner),
         }
     }
 
     pub(crate) fn from_string(private_key: String) -> Result<Self, BdkError> {
-        let descriptor_secret_key = BdkDescriptorSecretKey::from_str(private_key.as_str())
+        let inner = BdkDescriptorSecretKey::from_str(private_key.as_str())
             .map_err(|e| BdkError::Generic(e.to_string()))?;
         Ok(Self {
-            descriptor_secret_key_mutex: Mutex::new(descriptor_secret_key),
+            inner_mutex: Mutex::new(inner),
         })
     }
 
     pub(crate) fn derive(&self, path: Arc<DerivationPath>) -> Result<Arc<Self>, BdkError> {
         let secp = Secp256k1::new();
-        let descriptor_secret_key = self.descriptor_secret_key_mutex.lock().unwrap();
-        let path = path.derivation_path_mutex.lock().unwrap().deref().clone();
-        match descriptor_secret_key.deref() {
+        let inner = self.inner_mutex.lock().unwrap();
```

Comment:
This variable name should also stay as it was even though the inner field name was changed, for test readability sake.
```suggestion
        let descriptor_secret_key = self.inner_mutex.lock().unwrap();
```